js/src/apbct-public--2--public.js

```diff
@@ -30,6 +31,7 @@ const ctFunctionFirstKey = function output(event) {
 
 // run cron jobs
 cronFormsHandler(2000);
+sendBotDetectorLogToAltSessions(500);
```

Comment:
No need to send this ajax requests if the option `Use Anti-Spam by CleanTalk JavaScript library` is disabled.

Reply:
👍

Reply:
https://github.com/CleanTalk/wordpress-antispam/pull/509/commits/745a32454619fe68606cba10c4fff5704d25d508